test/e2e/e2e_test.go

```diff
@@ -672,7 +692,21 @@ var _ = Describe("e2e multiple external gateway update validation", func() {
 		jsonFlag := "-o=jsonpath='{.items..metadata.name}'"
 		fieldSelectorFlag := fmt.Sprintf("--field-selector=spec.nodeName=%s", ovnWorkerNode)
 		fieldSelectorHaFlag := fmt.Sprintf("--field-selector=spec.nodeName=%s", ovnHaWorkerNode)
-		// start the container that will act as an external gateway
+		ciNetworkName = "kind"
+		ciNetworkFlag = "{{ .NetworkSettings.Networks.kind.IPAddress }}"
+		// Determine which network bridge the kind install is using. KIND 7 and before use the default network name of 'bridge'
+		controlNodeIP, err := runCommand("docker", "inspect", "-f", ciNetworkFlag, ovnControlNode)
```

Comment:
just FYI, you can make it agnostic of the bridge name if you know that there is only one address per IP family in the container
```sh
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}},{{.GlobalIPv6Address}}{{end}}' ovn-control-pl
ane
172.18.0.4,fc00:f853:ccd:e793::4
aojea@elder:~/go/src/github.com/ovn-org/ovn-kubernetes$ docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ovn-control-plane
172.18.0.4
```

Reply:
Ty for the feedback. As we discussed in Slack we needed the network name for placing the new containers on the matching network as the CI network here: https://github.com/ovn-org/ovn-kubernetes/blob/c82d5759c6e1e6eab7911fce4e76ac0c0dc0b752/test/e2e/e2e_test.go#L558 

Ty!

Reply:
yup, you are absolutely right